aries-site/src/layouts/content/ExampleComponents.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button } from 'grommet';
+import { Subsection } from '.';
+import { SubsectionText } from '../../components';
+
+export const ExampleComponents = ({ components }) => (
+  <Subsection name="What components make this screen?" level={3}>
+    <SubsectionText>
+      To find more details about the usage of each of these components, check
+      out their documentation.
+    </SubsectionText>
+    <Box direction="row-responsive" gap="medium" wrap>
+      {components.map(component => (
```

Comment:
Inconsistent formatting of buttons on mobile, multiple devices\

![Screen Shot 2020-02-25 at 2 23 22 PM](https://user-images.githubusercontent.com/1756948/75288926-84292f00-57da-11ea-8137-c5cd64970a68.png)

Reply:
Good catch -- this is because Tiles doesn't have an href because we don't have documentation currently for it anywhere. I can point it to grid for now? -- when buttons have an href the text becomes left justified.

Reply:
We should have a section on the site for Tiles though, since it's an aries-core component that Shimi has already built out.